source/_docs/ecosystem/certificates/lets_encrypt.markdown

```diff
@@ -333,7 +333,6 @@ Now edit your `configuration.yaml` file to reflect the SSL entries and your base
 http:
   ssl_certificate: /etc/letsencrypt/live/examplehome.duckdns.org/fullchain.pem
   ssl_key: /etc/letsencrypt/live/examplehome.duckdns.org/privkey.pem
-  base_url: examplehome.duckdns.org
```

Comment:
The text above mentions base url. We should probabably add a line to change that in your config.

Reply:
Well spotted, thanks! Will update